build.gradle

```diff
@@ -22,17 +22,8 @@ plugins {
 
 conda {
     'miniconda2-build' {
-        packages.add('cmake')
-        pythonPackages.addAll('conan', 'grpcio-tools', 'twine')
-    }
-}
-
-tools {
```

Comment:
Is it intentional to delete the node section?

Reply:
Yeah - sorry for unrelated change, but updating curiostack allows us to remove it.